scipy/io/wavfile.py

```diff
@@ -304,7 +305,7 @@ class WAVE_FORMAT(IntEnum):
     DEVELOPMENT = 0xFFFF
 
 
-KNOWN_WAVE_FORMATS = {WAVE_FORMAT.PCM, WAVE_FORMAT.IEEE_FLOAT}
+KNOWN_WAVE_FORMATS = {WAVE_FORMAT.PCM, WAVE_FORMAT.IEEE_FLOAT, WAVE_FORMAT.MULAW }
```

Comment:
The addition of the new format means that `test_read_unknown_wave_format()` will fail because mulaw is now handled. Not sure if there's another known algorithm that could be substituted in to that unit test until we support it, or if we should just get rid of that test if all major formats are supported now. Could also just add a completely invalid format type I suppose.

Reply:
There are a long list of WAV formats that aren't supported by scipy, but MULAW and ALAW are interesting because they are both trivial enough to be implemented on an 8-bit microcontroller and compressed enough to transfer over a UART or almost any other serial device like an XBee or similar wireless IoT networks in realtime.